src/passes/references/externalReturnReceiver.ts

```diff
@@ -58,9 +58,9 @@ export class ExternalReturnReceiver extends ASTMapper {
 }
 
 function addOutputValidation(decl: VariableDeclaration, ast: AST) {
-  const validationFunctionCall = ast
-    .getUtilFuncGen(decl)
-    .boundChecks.inputCheck.gen(decl, safeGetNodeType(decl, ast.inference));
+  const type = safeGetNodeType(decl, ast.inference);
+  if (!checkableType(type)) return;
+  const validationFunctionCall = ast.getUtilFuncGen(decl).boundChecks.inputCheck.gen(decl, type);
```

Comment:
I think it's here by an accident, it's the code from #962 and would cause merge conflicts between develop and cairo-1.0